### DISCUSSION TITLE
Backspace button actual deleting the selected options.

### DISCUSSION THREAD

**Discussion by jpc0rrea** [koolamusic/chakra-ui-autocomplete#98] [Ideas]
When the user selects an option, and then deletes what he typed, if he continues to press the backspace key, the selected items begin to be deselected. I think a good new feature would be to cancel this behavior, and when the user continue to type backspace nothing really happens. Like if the only way to deselect a option is by pressing the "X" button on the created option tag.

**Comment by tenondra** [comment]
Hi, I have the same issue. Sadly there is no way to catch the backspace event too to prevent this right now.

**Comment by koolamusic** [comment]
This is an accessibility enhancement, The backspace is supposed to delete selections by design, unless you would be proposing a prop that will disable this default behaviour, of only allowing users to delete their inputs, however most users design their tags to be present in the input box, and in this case, the backspace simulates the behaviour of deleting the selected items even though they aren't primitive input values. 

So we need to think about this in a manner that is both accessible and controlled.
